src/inputs/AudioInput.js

```diff
@@ -1,13 +1,32 @@
 import AudioAnalyzer from './AudioAnalyzer'
 import { inputFired } from '../store/inputs/actions'
+import { uNodeCreate } from '../store/nodes/actions'
 
 export default (store) => {
   const gotStream = (stream) => {
     const input = new AudioAnalyzer(stream)
     const bandIds = ['audio_0', 'audio_1', 'audio_2', 'audio_3']
 
+    store.dispatch(uNodeCreate('audioNormalizeLevels', {
+      title: 'Normalize Levels',
+      type: 'param',
+      key: 'normalizeLevels',
+      value: input.normalizeLevels,
+      id: 'audioNormalizeLevels'
+    }))
+
+    store.dispatch(uNodeCreate('audioLevelsFalloff', {
+      title: 'Levels Falloff',
+      type: 'param',
+      key: 'levelsFalloff',
```

Comment:
I'm not sure if there's any need for `key` here but there's no harm in it being here. The `key` is used for params.